src/main/java/org/jabref/logic/importer/AdvancedFetcher.java

```diff
@@ -0,0 +1,38 @@
+package org.jabref.logic.importer;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+import org.jabref.logic.importer.fetcher.AdvancedSearchConfig;
+import org.jabref.logic.net.URLDownload;
+import org.jabref.model.entry.BibEntry;
+
+/**
+ * This interface allows SearchBasedParserFetcher fetchers to test their corresponding
+ * library APIs for their advanced search options, e.g. search in the "title" field.
+ */
+public interface AdvancedFetcher extends SearchBasedParserFetcher {
+
+    /**
+     * This method is used to send queries with advanced URL parameters.
+     * This method is necessary as the performSearch method does not support certain URL parameters that are used for
+     * fielded search, such as a title, author, or year parameter.
+     *
+     * @param advancedSearchConfig the search config defining all fielded search parameters
+     */
+    default List<BibEntry> performAdvancedSearch(AdvancedSearchConfig advancedSearchConfig) throws FetcherException {
```

Comment:
the `Advanced` in the name is not really necessary I guess.

Moreover, I would rename the paramaer to `ComplexQuery` (or `ComplexSearchQuery`) since this is not really a configuration.

Reply:
Thanks, I modified the names accordingly.